ui/src/app/views/homepage/component.html

```diff
@@ -1,139 +1,67 @@
 <div class="background-image">
-  <div class="row" style="display:flex; justify-content: center;flex-wrap: nowrap">
-    <div *ngFor="let card of cardDetails">
-      <div class="card double">
-        <div class="row">
-          <img class="icon topIcon" src={{card.icon}}>
-        </div>
-        <div class="row heading">
-          {{card.title}}
-        </div>
-        <div class="row">
-          <label class="text">{{card.text}}</label>
-        </div>
-        <div class="row button">
-          COMING SOON...
-        </div>
-      </div>
-    </div>
-  </div>
-  <div *ngIf="firstTimeUser">
-    <div class="row" style="display:flex; justify-content:center;">
-      <div class="card double">
-        <div class="row">
-          <img class="icon analyzeIcon" src="/assets/icons/Analyze.svg">
-        </div>
-        <div class="row analyzeHeading">Analyze and Publish with Workspaces
-        </div>
-        <div class="row">
-          <label class="text">Dolor sit amet consectetuer adipiscing sed diam
-            euismod tincidunt ut laoreet dolore.</label>
-        </div>
-        <div class="row">
-          <button class="btn btn-homepage" (click)="addWorkspace()">CREATE WORKSPACE</button>
+  <div class="row" style="display:flex; justify-content:center">
+    <div class="card single">
+      <div class="row" style="flex-direction: row; height: 17.47%; margin-bottom: 0.5rem; justify-content: flex-start; flex-wrap: nowrap;">
+        <div class="col main-heading">Researcher Workbench</div>
+        <div class="heading-links" style="display: flex; flex-wrap: nowrap; overflow: hidden;">
+          <a class="links" (click)="addWorkspace()">Create a
+            Workspace</a>
+          <a class="links" (click)="listWorkspaces()">See All
+            Workspaces</a>
+          <a class="links">Quick Tour</a>
```

Comment:
Is this functional yet? Otherwise I would probably remove it and let the person who adds quick tour add this.

Reply:
it's not.  i can comment it out with a note